src/main/java/de/leonhard/storage/internal/provider/MapProvider.java

```diff
@@ -1,16 +1,19 @@
 package de.leonhard.storage.internal.provider;
 
+import org.jetbrains.annotations.NotNull;
+
 import java.util.HashMap;
 import java.util.LinkedHashMap;
 import java.util.Map;
 
 public abstract class MapProvider {
 
-  public Map<String, Object> getMapImplementation() {
+  @SuppressWarnings("unused")
+  public @NotNull Map<String, Object> getMapImplementation() {
```

Comment:
Why does this need the unused mark? Is there any error in some other code, why this is not used or something?